### ISSUE TITLE
Gnuroot Bash is terminated when switching to another android app and back to the terminal emulator

### ISSUE THREAD

**Issue by ruelpsr** [corbinlc/gnuroot#39]
First of all, thanks for your work.
As I have written in the title, I have the folliwing problem on Android lollipop arm:
I launch a gnuroot wheezy bash from the gnuroot app. Terminal emulator turns up, i can run commands, all fine. I then switch to droidedit or google chrome to write a script/read some docs. Then I go back to the terminal emulator app, but the shell is gone. Seems like android has terminated the bash process because of memory load.
Is there a way to mark the bash process as an 'important' process that should not be killed but suspended to flash or kept in memory? That would be useful for me.
I think there must be a way as memory-wasting apps like whatsapp (needs 50MB) run all the time.

**Comment by corbinlc** [comment]
Inside the Android Terminal Emulator window that launches, can you go click on the "Take WakeLock" from within the settings?  Can you tell me if this helps?

**Comment by ruelpsr** [comment]
Hello Corbin!

Thanks for your response. The problem I reported seemed to be an Android Lollipop specific problem. The kernel leaked memory and at the same time does not seem to support paging. Instead processes are terminated when in background.
On my nexus7 this started out with gnuroot and Google chrome. later other processes were affected too. I had to reboot. Was all fine, then
If this happens again, I will try wakelock.
